prophetess/values.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
same question as earlier -- is there a reason for including the `---` here?

Reply:
This is just YAML garbage. It indicates a new "document" but is optional if you only ever have one YAML document. Copy pasta I'll update!